src/foam/dao/history/HistoryRecord.js

```diff
@@ -14,11 +14,15 @@ foam.CLASS({
   tableColumns: [
     'timestamp',
     'objectId',
-    'user',
+    'subject',
     'updates'
   ],
 
   properties: [
+    {
+      name: 'user',
+      getter: function() { return this.subject.user; }
+    },
```

Comment:
This is unnecessary since we're setting the user and agent info on create of history record.